tools/Dockerimage.crossbuild

```diff
@@ -0,0 +1,50 @@
+ARG AUTOWARE_DOCKER_ARCH
+ARG AUTOWARE_TARGET_ARCH
+FROM multiarch/alpine:${AUTOWARE_TARGET_ARCH}-latest-stable AS bootstrap
+
+FROM ${AUTOWARE_DOCKER_ARCH}/ros:dashing AS sysroot
+ARG AUTOWARE_TARGET_ARCH
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+COPY --from=bootstrap /usr/bin/qemu-${AUTOWARE_TARGET_ARCH}-static /usr/bin/qemu-${AUTOWARE_TARGET_ARCH}-static
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  libboost-all-dev \
+  libtinyxml2-dev \
+  ros-dashing-osrf-testing-tools-cpp \
+  ros-dashing-ament-cmake-nose \
+  ros-dashing-launch-testing
+
+# hadolint ignore=SC2156
+RUN find / -depth -xdev -type l -lname '/*' -exec sh -c 'linkpath="$(readlink {})" && rm -f {} && ln -frsv "$linkpath" "{}"' \;
+
+FROM ros:dashing AS builder
+ARG AUTOWARE_TARGET_ARCH
+ARG AUTOWARE_TARGET_PLATFORM
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+ENV AUTOWARE_SYSROOT /sysroot/${AUTOWARE_TARGET_PLATFORM}
+COPY --from=sysroot /lib ${AUTOWARE_SYSROOT}/lib
+COPY --from=sysroot /usr/include ${AUTOWARE_SYSROOT}/usr/include
+COPY --from=sysroot /usr/lib ${AUTOWARE_SYSROOT}/usr/lib
+COPY --from=sysroot /usr/share/pkgconfig ${AUTOWARE_SYSROOT}/usr/share/pkgconfig
+COPY --from=sysroot /opt ${AUTOWARE_SYSROOT}/opt
+COPY --from=sysroot /etc/alternatives ${AUTOWARE_SYSROOT}/etc/alternatives
+
+# Fix Bionic-specific issues
+# ignore pinning package versions for apt
+# hadolint ignore=DL3008
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    tzdata \
+    gnupg && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    gcc-aarch64-linux-gnu \
+    g++-aarch64-linux-gnu \
+    default-jre
+
+RUN find /sysroot/opt/ros/dashing/ -name "*.cmake" -type f -exec sed -i -e "s#/usr/lib#/sysroot/usr/lib#g" {} \;
```

Comment:
Does this mean that the resulting performance test binaries will have `/sysroot/...` hardcoded somewhere?

Reply:
This is only for the cmake files, so that when setting CMAKE_SYSROOT or CMAKE_FIND_ROOT_PATH and the modules are found, the paths returned for any libraries or binaries include the sysroot.